docker/docker-compose.1804.51.yaml

```diff
@@ -34,6 +34,9 @@ services:
       - MAX_ALLOCS_ALLOWED_modifying_byte_buffer_view=6010
       - MAX_ALLOCS_ALLOWED_schedule_10000_tasks=90050
       - MAX_ALLOCS_ALLOWED_read_10000_chunks_from_file=210050
+      - MAX_ALLOCS_udp_1000_reqs_1_conn=16250
+      - MAX_ALLOCS_ALLOWED_udp_1_reqs_1000_conn=224050
+      - SANITIZER_ARG=--sanitize=thread
```

Comment:
Remove this sanitizer thread and everything should pass :)

Reply:
Thanks - I messed up the merge.